docs/.wordlist.txt

```diff
@@ -34,3 +34,7 @@ ovs
 vswitchd
 github
 io
+st
+nd
+rd
+th
```

Comment:
@pmatulis `spellcheck` complained about my usage of `2nd`, `3rd` and `4th` so I added the suffixes to the wordlist, however I'm afraid that this can overshadow actual typos. Any suggestions on how to do this better?

Reply:
> This new material is great but we may be providing too much information to the user. I suggest we give less context of why we need to perform the steps. If you feel it to be important enough we can provide more detailed content in a Concepts page.

I understand your concern here but this specific doc is slightly special. It describes a situation that affects only people that installed `microovn` from stable channel 'till this day. Anyone who installed it from `edge` or anyone installing it from (default) stable channel starting today, will have have this (TLS) feature enabled automatically. These manual transition steps are really only necessary for small number of current early adopters, but we wanted to give them instructions on how to move on to current stable channel. That's why I think that moving this to the "Concepts page" might not be a good idea.

The reason I went into details here is that there's a lot of repetition in this process. You need to perform 5 steps three times for two clusters and the parameters for these steps vary slightly (almost) each time. That's why my approach was to write these 5 general steps with enough detail that the user will understand what these steps do and how they should be altered each time they are performed.

Reply:
True, this is only for users on an older snap revision and so wouldn't be appropriate in a Concepts section.

Alright, let's keep it.